.github/workflows/lint-and-tests.yml

```diff
@@ -49,13 +50,13 @@ jobs:
         run: echo "turbo_args=--force=true" >> "$GITHUB_OUTPUT"
 
   lint:
-    # This Job should run either on `merge_groups` or `push` events
+    # This Job should run either on `merge_groups`, `push`, or `workflow_dispatch` events
```

Comment:
What's the reason for the changes here? 🤔

Reply:
Since this PR updated the workflow for the new package, I added a workflow_dispatch trigger to run the modified workflow manually.